README.md

```diff
@@ -43,33 +37,36 @@ Haveno is a community-driven project. For it to be succesful it's fundamental to
 
 Temporary email: havenodex@protonmail.com
 
-## FAQ
+### Styling guidelines
+
+See [CONTRIBUTING.md](docs/CONTRIBUTING.md)
+
+## Bounties
 
-See the [FAQ in the wiki](https://github.com/haveno-dex/haveno/wiki/FAQ).
+To incentivize development and reward contributors we adopt a simple bounty system. Contributors may be awarded bounties after completing a task (resolving an issue). [More details in the docs](docs/bounties.md).
 
 ## Running a local Haveno test network
 
 1. Download [Monero CLI](https://www.getmonero.org/downloads/) for your system and sync Monero stagenet: `./monerod --stagenet --rpc-login superuser:abctesting123`, or alternatively, [set up a local Monero stagenet network](#running-a-local-monero-stagenet-network) (recommended)
-3. Download and install [Bitcoin-Qt](https://bitcoin.org/en/download)
```

Comment:
This is #2 on master.  Is your branch up-to-date?

Reply:
It is. My last commit is 8756d8feca8d55a70470e92d38d4cd1c7021b226. Weird.

Reply:
So, apparently GitHub is autocorrecting the list, but it's wrong in the readme. This PR restores the correct order.